source/system-administrators/security/configure-headers-based-auth.rst

```diff
@@ -140,4 +140,8 @@ Let's take a look at an example setup for headers based authentication using SAM
 
 Once a user has been authenticated, the user will be granted access to Studio.  If the user is not yet in the Studio database, the user will be created in the database with the attributes from the header, and also added to the groups of the site specified in the header.
 
+.. note::
+
+   The **secure_key** header value set in the ``auth_mellon configuration`` (Item number 5 above) should match the value listed in the ``studio-config-override.yaml`` for the property **studio.authentication.headers.secureKeyHeaderName**.  Also, remember to change these default values.
```

Comment:
What does Also, remember to change these default values mean? In every installation does it have to be different? Like a random generated text?

Reply:
Hi! We're thinking of change the note to the following (Let me know if this note explains it better? Thanks!): 
```
The **secure_key** header value set in the ``auth_mellon configuration`` (Item number 5 above) should match the value listed in the ``studio-config-override.yaml`` for the property **studio.authentication.headers.secureKeyHeaderValue**.  This becomes a handshake between Studio and HTTPd and protects your installation from someone potentially faking headers. You should change the default to some arbitrary value to better protect your installation.
```